browser/extensions/brave_extension_provider.cc

```diff
@@ -47,43 +47,48 @@ bool BraveExtensionProvider::IsVetted(const Extension* extension) {
   // unit tests.
   // Don't add new extensions to this list. Add them to
   // the files managed by the extension whitelist service.
+  return BraveExtensionProvider::IsVetted(extension->id());
+}
+
+bool BraveExtensionProvider::IsVetted(const std::string id) {
   static std::vector<std::string> vetted_extensions({
-    brave_extension_id,
-    brave_rewards_extension_id,
-    brave_sync_extension_id,
-    brave_webtorrent_extension_id,
-    pdfjs_extension_id,
-    // Web Store
-    "ahfgeienlihckogmohjhadlkjgocpleb",
-    // Brave Automation Extension
-    "aapnijgdinlhnhlmodcfapnahmbfebeb",
-    // Test ID: Brave Default Ad Block Updater
-    "naccapggpomhlhoifnlebfoocegenbol",
-    // Test ID: Brave Regional Ad Block Updater
-    // (9852EFC4-99E4-4F2D-A915-9C3196C7A1DE)
-    "dlpmaigjliompnelofkljgcmlenklieh",
-    // Test ID: Brave Tracking Protection Updater
-    "eclbkhjphkhalklhipiicaldjbnhdfkc",
-    // Test ID: PDFJS
-    "kpbdcmcgkedhpbcpfndimofjnefgjidd",
-    // Test ID: Brave HTTPS Everywhere Updater
-    "bhlmpjhncoojbkemjkeppfahkglffilp",
-    // Test ID: Brave Tor Client Updater
-    "ngicbhhaldfdgmjhilmnleppfpmkgbbk",
+      brave_extension_id,
+      brave_rewards_extension_id,
+      brave_sync_extension_id,
+      brave_webtorrent_extension_id,
+      crl_set_extension_id,
+      pdfjs_extension_id,
+      hangouts_extension_id,
+      widevine_extension_id,
+      // Web Store
+      "ahfgeienlihckogmohjhadlkjgocpleb",
+      // Brave Automation Extension
+      "aapnijgdinlhnhlmodcfapnahmbfebeb",
+      // Test ID: Brave Default Ad Block Updater
+      "naccapggpomhlhoifnlebfoocegenbol",
+      // Test ID: Brave Regional Ad Block Updater
+      // (9852EFC4-99E4-4F2D-A915-9C3196C7A1DE)
+      "dlpmaigjliompnelofkljgcmlenklieh",
+      // Test ID: Brave Tracking Protection Updater
+      "eclbkhjphkhalklhipiicaldjbnhdfkc",
+      // Test ID: PDFJS
+      "kpbdcmcgkedhpbcpfndimofjnefgjidd",
+      // Test ID: Brave HTTPS Everywhere Updater
+      "bhlmpjhncoojbkemjkeppfahkglffilp",
+      // Test ID: Brave Tor Client Updater
+      "ngicbhhaldfdgmjhilmnleppfpmkgbbk",
```

Comment:
@pilgrim-brave We will be using `IsVetted` to allow updates for components that Brave Supports. I think we should move the `ID` for `local-data-files` to prevent any update issues for other components if the `local-data-files` gets corrupted.

Reply:
OK